performance_manager/tests/test_postgres.py

```diff
@@ -30,21 +76,332 @@ def set_environment_variables() -> Iterator[None]:
+
+        if isinstance(filename, list):
+            to_load = filename
+        else:
+            to_load = [filename]
+
+        if len(to_load) == 0:
+            return pyarrow.Table.from_pydict({})
+
+        return parquet.ParquetDataset(
+            to_load, filesystem=active_fs, filters=filters
+        ).read_pandas()
+
+    monkeypatch.setattr(
+        "lib.s3_utils._get_pyarrow_table", mock__get_pyarrow_table
+    )
     yield
 
 
-def test_environ() -> None:
-    """simple test stub"""
-    # ensure that the environment has been stook up correctly.
-    assert int(os.environ.get("BOOTSTRAPPED", 0)) == 1
+def check_logs(caplog: pytest.LogCaptureFixture) -> None:
+    """
+    utility to check that all logged statements were properly formatted, no
+    errors were reported, and that all processes had a log start and log complete.
+    """
+
+    def message_to_dict(message: str) -> Dict[str, str]:
+        """convert our log messages into dict of properties"""
+        message_dict = {}
+
+        for part in message.replace(",", "").split(" "):
+            try:
+                (key, value) = part.split("=")
+                message_dict[key] = value
+            except Exception as exception:
+                pytest.fail(
+                    f"Unable to parse log message {message}. Reason {exception}"
+                )
+        return message_dict
 
-    db_manager = DatabaseManager()
+    # keep track of logged processes to ensure order is correct
+    process_stack: List[Dict[str, str]] = []
+
+    for record in caplog.records:
+        # if the record was an Error Level, fail immediately
+        if record.levelname == "ERROR":
+            pytest.fail(f"Error messages encountered {record.message}")
+
+        # skip debug level logs
+        if record.levelname == "DEBUG":
+            continue
+
+        log = message_to_dict(record.message)
+
+        # check for keys that must be present
+        must_keys = ["status", "uuid", "parent", "process_id", "process_name"]
+        for key in must_keys:
+            if key not in log:
+                pytest.fail(f"Log missing {key} key - {record.message}")
+
+        # process the record, ensuring proper ordering
+        if log["status"] == "complete":
+            # process should be at the end of the stack
+            if process_stack[-1]["uuid"] != log["uuid"]:
+                pytest.fail(
+                    f"Improper Ordering of Log Statements {caplog.text}"
+                )
+            if "duration" not in log:
+                pytest.fail(f"Log missing duration key - {record.message}")
+            process_stack.pop()
+        elif log["status"] == "started":
+            process_stack.append(log)
+
+    if len(process_stack) != 0:
+        pytest.fail(f"Not all processes logged completion {caplog.text}")
 
-    paths_to_load = get_unprocessed_files(
-        "RT_VEHICLE_POSITIONS", db_manager, file_limit=6
+
+def test_static_tables(
```

Comment:
test that static data can be processed correctly.

this test also seeds the db with static info we'll need for gtfs_rt processing.